src/js/modules/breakpointHelper.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @overview Breakpoint handler
+ *
+ * @module breakpointHelper
+ */
+
+export const getBreakpoints = () => {
+    const output = {};
+
+    const screenSizer = document.createElement('div');
+    screenSizer.classList.add('c-screen-sizer');
+
+    document.body.appendChild(screenSizer);
+
+    const breakpoints = window.getComputedStyle(document.querySelector('.c-screen-sizer'))
+        .getPropertyValue('content')
+        .replace(/"/g, '')
+        .replace(/'/g, '')
+        .split(',');
+
+    if (breakpoints.length === 1 && breakpoints[0] === '') {
+        // When there is no content
+        return output;
+    }
+
+    breakpoints.forEach(breakpoint => {
```

Comment:
Can use `reduce()` and return the value here:

```js
return breakpoints.reduce((prev, current) => {
    const [breakpointName, breakpointValue] = current.split(':');
    prev[breakpointName] = breakpointValue;
    return prev;
}, {});
```

Reply:
I'd argue that's actually harder to interpret than a simple for-each loop.

Reply:
It's OK for now but I think we should be pushing to use functions like these a bit more in the codebase, adding a short comment could explain what the `reduce` does for anyone unfamiliar?